tests/ui-testing/playwright-tests/dashboards/dashboardFilter.spec.js

```
@@ -0,0 +1,586 @@
+    state: "hidden",
+  });
+  await dashboardListApi;
+  await page.waitForTimeout(500);
+}
+
+test.describe("dashboard UI testcases", () => {
+  // let logData;
+  function removeUTFCharacters(text) {
+    // console.log(text, "tex");
+    // Remove UTF characters using regular expression
+    return text.replace(/[^\x00-\x7F]/g, " ");
```

Biome

[error] Unexpected control character in a regular expression.

Control characters are unusual and potentially incorrect inputs, so they are disallowed.

(lint/suspicious/noControlCharactersInRegex)